modelscope/models/nlp/T5/backbone.py

```diff
@@ -24,6 +24,8 @@
 from torch import nn
 from torch.utils.checkpoint import checkpoint
 from transformers.activations import ACT2FN
+from transformers.modeling_outputs import \
+    BaseModelOutputWithPastAndCrossAttentions
```

Comment:
Is this compatible with previous old version of transformers?

Reply:
Sure, This changing is to use transformers' output instead of mslib's output class, which can be imported since the lowest version of transformers in mslib's requirement file.